packages/core/content-manager/admin/src/pages/EditView/components/FormInputs/Relations/Relations.tsx

```diff
@@ -1537,7 +1557,9 @@ const UnstableListItem = ({ data, index, style }: ListItemProps) => {
                 <RelationModal
                   open={isModalOpen}
                   onToggle={() => {
-                    setIsModalOpen(!isModalOpen);
+                    setIsModalOpen(false);
+                    setContextIsModalOpen(false);
```

Comment:
We already discussed on the other PR that we aren't going to manage two states like this, struggling to understand why you want to re-introduce that idea.